test/db/anal/x86_64

```diff
@@ -3474,3 +3474,85 @@ EOF
 EXPECT=<<EOF
 EOF
 RUN
+
+NAME=lea pseudo improvements (pseudo disabled)
+FILE=-
+CMDS=<<EOF
+e asm.arch=x86
+e asm.bits=64
+e asm.comments=false
+e asm.pseudo=false
+wx 544889e5b40266b83333b85555444448b8bebafecaefbeadde4889d8488d03488b4308488d4308488d041b488d441b0a488d430a488d44240c488d450b488d45f5488d442404488d4508488d45f8
+aa
+pd 19
+EOF
+EXPECT=<<EOF
+/ 512: fcn.00000000 (int64_t arg_8h, int64_t arg_bh, int64_t arg_ch);
+|           ; var int64_t var_bh @ rbp-0xb
+|           ; var int64_t var_8h @ rbp-0x8
+|           ; arg int64_t arg_8h @ rbp+0x8
+|           ; arg int64_t arg_bh @ rbp+0xb
+|           ; var int64_t var_4h @ rsp+0x4
+|           ; arg int64_t arg_ch @ rsp+0xc
+|           0x00000000      54             push rsp
+|           0x00000001      4889e5         mov rbp, rsp
+|           0x00000004      b402           mov ah, 2
+|           0x00000006      66b83333       mov ax, 0x3333
+|           0x0000000a      b855554444     mov eax, 0x44445555
+|           0x0000000f      48b8bebafeca.  movabs rax, 0xdeadbeefcafebabe
```

Comment:
Could you add the `movabs` op in the `ops` array in `replace` function? It should be just a matter of adding `{ "movabs", "# = #", {1, 2}},` to the array. I know it's not strictly related to your change, but at least we can create a correct test ;)